Assignment-1/requests.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+/**
+ * API call response.
+ *
+ * @param string $url
+ *   URL of API call.
+ *
+ * @return mixed
+ *   Returns API response in PHP associative array form.
+ *   NULL in case of any exception.
+ */
+function request(string $url) : mixed {
+  try {
+    $client = new \GuzzleHttp\Client();
+    $response = $client->request('GET', $url);
+    // Convert JSON to PHP Associative array and return.
+    return json_decode($response->getBody(), TRUE);
```

Comment:
Do you always get JSON response if you hit any URL?

Reply:
Added checks for response in JSON or not.